### ISSUE TITLE
It seems this still needs a list of apps declared in Info.plist

### ISSUE THREAD

**Issue by Naphier** [rbaulin/CanOpenURL#1]
Is this true? Do the URLs need to be added to the info.plist? Can I simply feed a list of bundle IDs into (BOOL)IsAppInstalled? Just trying to double check before we try to use this to repair our SDK. Thanks!

**Comment by rbaulin** [comment]
@Naphier no, you don't need to add anything to Info.plist, that's the point. Check out example project.

**Comment by Naphier** [comment]
Thanks, man. Just wanted to make entirely sure. You rock for setting this up!

**Comment by rbaulin** [comment]
@Naphier thanks! :point_left:
